addon/components/bsy-button/close.js

```diff
@@ -3,10 +3,11 @@ import layout from '../../templates/components/bsy-button/close';
 
 export default Component.extend({
   layout,
-  tagName: "button",
-  classNames: "btn btn-secondary-inverse btn-round",
 
-  click: function() {
-    this.$().prop('disabled', 'disabled');
+  tagName: 'button',
+  classNames: 'btn btn-secondary-inverse btn-round',
+
+  click() {
+    this.element.disabled = true;
```

Comment:
This could be handled by `attributeBindings`.

Reply:
Yes, but in this case we need to introduce another attribute.

Reply:
We can have it as is, but if I'd written the code I would've gone the Ember way. :+1:

Reply:
Agree, still room for a lot of improvement. This addon (the ember part) will eventually be faded out though.